.github/workflows/pypi_packaging.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish to PyPi
+
+# Controls when the action will run.
+# Triggers the workflow on push or pull request events
+on: [push]
```

Comment:
This needs to be changed so that it only runs when pushed to cromsell2.0. 
keeping it as is for now so that I can test this action once the pypi token is added to git secrets

Reply:
Keep this manual. 
use testpypi as a dev release

Reply:
Will change to [workflow_dispatch](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) once ready for merging.